themes/src/main/resources-community/theme/base/account/messages/messages_nl.properties

```
@@ -30,16 +54,33 @@ region=Staat, provincie of regio
 postal_code=Postcode
 country=Land
 emailVerified=E-mailadres geverifieerd
+website=Web pagina
```

I think that 'Website' has become a more common nomenclature.
```suggestion
website=Website
```